ppr-ui/src/views/Dashboard.vue

```diff
@@ -96,12 +119,18 @@ import {
   ManufacturedHomeSearchResponseIF, // eslint-disable-line no-unused-vars
   SearchResponseIF // eslint-disable-line no-unused-vars
 } from '@/interfaces'
-import { useAuth, useNavigation } from '@/composables'
+import { useAuth, useNavigation, useUserAccess } from '@/composables'
 
 export default defineComponent({
   name: 'Dashboard',
+  computed: {
+    ProductStatus () {
```

Comment:
What does this do?

Reply:
Since its read only (an enum in this case), this is instead of explicitly defining/returning the enum to use in the Template. 
I'm impartial on how we do it, this seems to be neat for when we don't use the enum in the script and only the template.
This is also how my editor auto imports the enums but that doesn't mean it's right 😆

Reply:
I like it, I think its pretty straightforward, and makes the script smaller which is great!